lib/matplotlib/contour.py

```diff
@@ -1296,10 +1297,14 @@ def _process_linewidths(self):
     def _process_linestyles(self):
         linestyles = self.linestyles
         Nlev = len(self.levels)
+        neg_ls = self.negative_linestyles
         if linestyles is None:
             tlinestyles = ['solid'] * Nlev
             if self.monochrome:
-                neg_ls = mpl.rcParams['contour.negative_linestyle']
+                # If negative_linestyles was not defined as a kwarg,
+                # define negative_linestyles with rcParams
+                if not neg_ls:
```

Comment:
```suggestion
                if neg_ls is None:
```
`negative_linestyle=''` would otherwise lead to surprising results.

Should you also set `self.negative_linestyles` here? I think it can make sense to not use the `neg_ls` local variable and just use `self.negative_linestyles` throughout the code here.

Reply:
In that case, I'll move the case with `rcParams['contour.negative_linestyle']` to where `self.negative_linestyles` is initially defined.